ajax/libs/slim-select/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "slim-select",
+  "description": "Slim vanilla javascript select dropdown",
+  "author": "Brian Voelker <brian@webiswhatido.com> (http://webiswhatido.com)",
+  "homepage": "https://slimselectjs.com",
```

Comment:
Use ssl protocol is better.
But I found there is a Certificate Error at this website.
There are issues with the site's certificate chain (net::ERR_CERT_COMMON_NAME_INVALID).
And this problem make Google Chrome show it is not secure.

I think there are two different choice can let it better:
1. Fix the site's certificate chain, let the website in secure.
2. Let's use `"homepage": "http://slimselectjs.com",`